### PR TITLE
Purge EdgeCast after deploying to production

### PR DESCRIPTION
**Pull request by joanniclaborde**

### CODE REVIEW

File: config/deploy-production.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    method: 'put',
+    url: 'https://api.edgecast.com/v2/mcc/customers/08DF/edge/purge',
+    headers: {
+      Authorization: 'TOK:' + edgeCastApiToken,
+      Host: 'api.edgecast.com',
+      Accept: 'application/json'
+    },
+    body: function() {
+      return {
+        MediaPath: 'http://www.' + projectName.replace(/^www\./, '') + '/',
```
Comment (pushred) [inline config/deploy-production.js]: Would be better to pull this from the package config like [we do in qa-deployer](https://github.com/SparkartGroupInc/qa-deployer/pull/11). There's backstagewithjon.bonjovi.com and idol.keithurban.net for instance.

Comment (joanniclaborde) [inline config/deploy-production.js]: `projectName` is already extracted from `package.json` (see line 4). If all our sites follow the `subdomain.domain.ext` pattern, then I could add `www` only if there's no subdomain. Looking at our edge CNAMEs on EdgeCast, that seems to be the case, except for some dev stuff like `staging.proxy.storyteller.io`.

Comment (pushred) [inline config/deploy-production.js]: Gotcha, yeah Dyn doesn't allow us to use CNAME records at the apex/root domain, so a subdomain is a requirement for any projects we host DNS for at least.

**Comment by pushred** [comment]
:+1:

**Pull request merged**